webpack.config.js

```diff
@@ -102,7 +102,7 @@ const commonConfig = {
         extensions: ['.tsx', '.ts', '.js', '.json'],
         // axe-core invokes require('crypto'), but only in a path we don't use, so we don't need a polyfill
         // See https://github.com/dequelabs/axe-core/issues/2873
-        fallback: { crypto: false },
+        fallback: { crypto: require.resolve('crypto-browserify'), stream: false },
```

Comment:
This should become unnecessary once #5388 merges (axe-core removed their crypto dependency, so the whole `fallback` config goes away in that PR)

Reply:
(I've just merged #5388 to make it easier to resolve this)